modules/docs/sc_param.md

```diff
@@ -36,76 +36,76 @@ The sc_param module provides methods to help you handle parameters for your stre
+| skip_nil_id                             | number | 1                                                                             | filter out events if their ID is nil (use 0 to accept them. YOU SHOULDN'T DO THAT)                                                                                                                                                                                                                                                            | host_status(neb), service_status(neb), ba_status(bam), acknowledgement(neb)                                                                                                                                                                                                              |                                                                                                                                                                                                                                                                                                  |
+| enable_bam_host                         | number | 0                                                                             | filter out events if the host name matches "^_Module_BAM_*" (which is a fake host related to Business Activities from the Centreon BAM module) (use 1 to accept them). Those are very special events with hostname being _Module_BAM_1 and service name being ba_x where "x" is the ID of the business activity. Even if this parameter only applies to service_status(neb) events, you will need to use the **NEB and BAM** filter in your stream connector configuration (otherwise you won't have the BAM cache built by the Centreon broker and your event will not be sent)                               | service_status(neb)                                                                                                                                                                                                                                                                      |                                                                                                                                                                                                                                                                                                  |
```

Comment:
```suggestion
| enable_bam_host                         | number | 0                                                                             | filter out events if the host name matches "^_Module_BAM_*" (which is a fake host related to Business Activities from the Centreon BAM module) (use 1 to accept them). Those are very special events with the hostname being _Module_BAM_1 and the service name being ba_x where "x" is the ID of the business activity. Even if this parameter only applies to service_status(neb) events, you will need to use the **NEB and BAM** filter in your stream connector configuration (otherwise you won't have the BAM cache built by Centreon Broker and your event will not be sent)                               | service_status(neb)                                                                                                                                                                                                                                                                      |                                                                                                                                                                                                                                                                                                  |
```